# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/glibc23/make_shlib_link.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makerules.orig	2002-10-12 11:52:02.000000000 +0200
+++ ./Makerules	2002-10-12 11:55:26.000000000 +0200
@@ -820,8 +820,7 @@
 endif
 ifndef make-shlib-link
 define make-shlib-link
-rm -f $@
-$(LN_S) $(<F) $@
+ln -sf $(<F) $@
 endef
 endif
 
--- ./Makefile.orig	2002-10-13 16:13:47.000000000 +0200
+++ ./Makefile	2002-10-13 16:16:26.000000000 +0200
@@ -110,7 +110,8 @@
 install: install-symbolic-link
 .PHONY: install-symbolic-link
 install-symbolic-link: subdir_install
-	$(symbolic-link-prog) $(symbolic-link-list)
+	while read from to ; do \
+		ln -sf $$from $$to ; done < $(symbolic-link-list)
 	rm -f $(symbolic-link-list)
 
 install:
